lib/msf/core/module/alert.rb

```diff
@@ -201,19 +242,41 @@ def add_alert(level, msg, &block)
   # with this method will not be displayed again.
   def alert_user
     self.you_have_been_warned ||= {}
+    without_prompt do
+      errors.each do |msg|
+        if msg && !self.you_have_been_warned[msg.hash]
+          print_error(msg)
+          self.you_have_been_warned[msg.hash] = true
+        end
+      end
 
-    errors.each do |msg|
-      if msg && !self.you_have_been_warned[msg.hash]
-        print_error(msg)
-        self.you_have_been_warned[msg.hash] = true
+      warnings.each do |msg|
+        if msg && !self.you_have_been_warned[msg.hash]
+          print_warning(msg)
+          self.you_have_been_warned[msg.hash] = true
+        end
       end
-    end
 
-    warnings.each do |msg|
-      if msg && !self.you_have_been_warned[msg.hash]
-        print_warning(msg)
-        self.you_have_been_warned[msg.hash] = true
+      infos.each do |msg|
+        if msg && !self.you_have_been_warned[msg.hash]
+          # Make prefix an empty string to avoid adding clutter (timestamps, rhost, rport, etc.) to the output
+          print_status(msg, prefix: '')
```

Comment:
I'm assuming that all of the calls to `print_*` shouldn't have a prefix?

Reply:
Seems like this might be leaving around tech debt/inconsistencies for the next person 👀

Reply:
I don't mind making the change, can sort that now